diff --git a/.env b/.env
index 5cbee27..81670e9 100644
--- a/.env
+++ b/.env
@@ -3,4 +3,4 @@ IMAGE_REPO=zammad/zammad-docker-compose
 POSTGRES_PASS=zammad
 POSTGRES_USER=zammad
 RESTART=always
-VERSION=-3.2.0-13
+VERSION=-3.3.0-6
diff --git a/Dockerfile b/Dockerfile
index b5001d1..9e824ad 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -8,5 +8,5 @@ LABEL org.label-schema.build-date="$BUILD_DATE" \
       org.label-schema.vcs-url="https://github.com/zammad/zammad" \
       org.label-schema.vcs-type="Git" \
       org.label-schema.vendor="Zammad" \
-      org.label-schema.schema-version="3.2.0" \
+      org.label-schema.schema-version="3.3.0" \
       org.label-schema.docker.cmd="sysctl -w vm.max_map_count=262144;docker-compose up"
diff --git a/README.md b/README.md
index 28eb778..8604c73 100644
--- a/README.md
+++ b/README.md
@@ -42,7 +42,7 @@ Like this, you can add your `docker-compose.prod.yml` to a branch of your Git re
 
 ## Upgrading
 
-### From =< 3.2.0-12
+### From =< 3.3.0-12
 
 We've updated the Elasticsearch image from 5.6 to 7.6. 
 As there is no direct upgrade path we have to delete all Elasticsearch indicies and rebuild them.
diff --git a/containers/zammad-elasticsearch/Dockerfile b/containers/zammad-elasticsearch/Dockerfile
index 4c105d2..f20aceb 100644
--- a/containers/zammad-elasticsearch/Dockerfile
+++ b/containers/zammad-elasticsearch/Dockerfile
@@ -9,7 +9,7 @@ LABEL org.label-schema.build-date="$BUILD_DATE" \
       org.label-schema.vcs-url="https://github.com/zammad/zammad" \
       org.label-schema.vcs-type="Git" \
       org.label-schema.vendor="Zammad" \
-      org.label-schema.schema-version="3.2.0" \
+      org.label-schema.schema-version="3.3.0" \
       org.label-schema.docker.cmd="sysctl -w vm.max_map_count=262144;docker-compose up"
 
 SHELL ["/bin/bash", "-e", "-o", "pipefail", "-c"]
diff --git a/containers/zammad-postgresql/Dockerfile b/containers/zammad-postgresql/Dockerfile
index bacf81c..3904638 100644
--- a/containers/zammad-postgresql/Dockerfile
+++ b/containers/zammad-postgresql/Dockerfile
@@ -10,7 +10,7 @@ LABEL org.label-schema.build-date="$BUILD_DATE" \
       org.label-schema.vcs-url="https://github.com/zammad/zammad" \
       org.label-schema.vcs-type="Git" \
       org.label-schema.vendor="Zammad" \
-      org.label-schema.schema-version="3.2.0" \
+      org.label-schema.schema-version="3.3.0" \
       org.label-schema.docker.cmd="sysctl -w vm.max_map_count=262144;docker-compose up"
 
 SHELL ["/bin/bash", "-e", "-o", "pipefail", "-c"]
diff --git a/containers/zammad/Dockerfile b/containers/zammad/Dockerfile
index 8fb92f5..40ae4be 100644
--- a/containers/zammad/Dockerfile
+++ b/containers/zammad/Dockerfile
@@ -37,7 +37,7 @@ LABEL org.label-schema.build-date="$BUILD_DATE" \
       org.label-schema.vcs-url="https://github.com/zammad/zammad" \
       org.label-schema.vcs-type="Git" \
       org.label-schema.vendor="Zammad" \
-      org.label-schema.schema-version="3.2.0" \
+      org.label-schema.schema-version="3.3.0" \
       org.label-schema.docker.cmd="sysctl -w vm.max_map_count=262144;docker-compose up"
 
 ENV GIT_BRANCH stable
